pkg/interface/src/views/landscape/components/Home/Post/PostItem/PostContent.tsx

```diff
@@ -1,17 +1,20 @@
 import { Col, ColProps } from '@tlon/indigo-react';
 import { Post } from '@urbit/api';
-import React, { ReactElement } from 'react';
-import styled from 'styled-components';
+import React, { ReactElement, useCallback, useState } from 'react';
+import styled, { css } from 'styled-components';
 import { GraphContent } from '~/views/landscape/components/Graph/GraphContent';
 
 type TruncateProps = ColProps & {
-  truncate?: number;
+  truncate: boolean;
 }
 
 const TruncatedBox = styled(Col)<TruncateProps>`
   display: -webkit-box;
-  -webkit-line-clamp: ${p => p.truncate ?? 'unset'};
   -webkit-box-orient: vertical;
+  ${p => p.truncate && css`
+    max-height: 300px;
+    mask-image: linear-gradient(to bottom, rgba(0,0,0,1) 0%, rgba(0,0,0,1) 60%, transparent 100%);
```

Comment:
It does because this is a mask you don't actually render this color. The black is only used to say what should show through the "mask" and then otherwise what's transparent gets masked away.

Reply:
He's right - it masks appropriately